neuralprophet/time_dataset.py

```diff
@@ -707,19 +673,14 @@ def sort_regressor_names(self, config):
         return additive_regressors_names, multiplicative_regressors_names
 
     def get_sample_targets(self, df_tensors, origin_index, n_forecasts, max_lags, predict_mode):
```

Comment:
This is no longer used, so it can be removed, right?
same for all other `get_sample_xyz`?